tests/suites/test_suite_psa_crypto_metadata.data

```diff
@@ -170,6 +170,18 @@ Asymmetric signature: SHA-256 + deterministic ECDSA using SHA-256
 depends_on:PSA_WANT_ALG_DETERMINISTIC_ECDSA:PSA_WANT_ALG_SHA_256
 asymmetric_signature_algorithm:PSA_ALG_DETERMINISTIC_ECDSA( PSA_ALG_SHA_256 ):ALG_IS_ECDSA | ALG_IS_DETERMINISTIC_ECDSA | ALG_ECDSA_IS_DETERMINISTIC | ALG_IS_HASH_AND_SIGN
 
+Asymmetric signature: pure EdDSA
+#depends_on:PSA_WANT_ALG_EDDSA
```

Comment:
Question: is it intentional for that line to be commented? (Same for the next two tests.)

Reply:
This doesn't look right. The metadata test suite does not assume that all metadata is correct if an algorithm is not supported (for example, size macros could be out of range), hence the dependency. The metadata test suite also should avoid having tests that are not executed in any configuration because we have a medium-term goal of having the CI enforce that, which would point towards not having these entries at all. I don't remember why I would have a commented-out dependency line, however. I'll give it a think and likely uncomment the dependencies.